apps/browser-extension-wallet/src/views/browser-view/features/assets/components/AssetsPortfolio/AssetsPortfolio.tsx

```diff
@@ -35,6 +38,16 @@ export interface AssetsPortfolioProps {
   isLoadingFirstTime?: boolean;
 }
 
+const searchTokens = (data: IAssetDetails[] | IRow[], searchValue: string) => {
+  const fields = ['name', 'policyId', 'fingerprint']; // Fields to search
```

Comment:
Two suggestions here. Consider removing the comment because IMO it is obvious. Try adding `as const` at the end so then it helps TS recognize those values can be used as keys for the data.

```suggestion
  const fields = ['name', 'policyId', 'fingerprint'] as const;
```

Reply:
Fixed on 55a094f